src/overlays/actors/ovl_Mir_Ray2/z_mir_ray2.c

```diff
@@ -15,7 +15,9 @@ void MirRay2_Destroy(Actor* thisx, PlayState* play);
 void MirRay2_Update(Actor* thisx, PlayState* play);
 void MirRay2_Draw(Actor* thisx, PlayState* play);
 
-#if 0
+void func_80AF3F70(MirRay2* this);
+void func_80AF3FE0(MirRay2* this, PlayState* play);
```

Comment:
Neither of these forward declarations is needed, we don't put them here unless they are.